src/main/java/org/broadinstitute/hellbender/tools/ConvertHeaderlessHadoopBamShardToBam.java

```diff
@@ -27,7 +27,7 @@
 @CommandLineProgramProperties(
         summary = "Convert a headerless hadoop bam shard into a readable bam",
         oneLineSummary = "Convert a headerless hadoop bam shard into a readable bam",
-        programGroup = TestSparkProgramGroup.class
+        programGroup = OtherProgramGroup.class
```

Comment:
Why is this in `OtherProgramGroup`? Did comms just not understand what this tool does? It is a diagnostic/debugging tool for allowing intermediate reads data on Spark to be read using conventional programs like `samtools`

Reply:
Should also probably be marked as `Experimental`

Reply:
The doc portion is cryptic and no, it's not clear what this tool does from the current documentation. I asked that the description be updated a few days ago.

Reply:
I'll mark as experimental. The only other group that makes sense is reads, unless we invent something new.  I'll move to the reads group for now.

Reply:
I don't like this move. We can discuss after the release. My quip is that then CreateHadoopBamSplittingIndex needs to move with ConvertHeaderlessHadoopBamShardToBam.